src/storage/mongodb/MongoDBStorage.ts

```diff
@@ -268,6 +269,15 @@ export default class MongoDBStorage {
     // Locks
     await this.handleIndexesInCollection(Constants.DEFAULT_TENANT, 'locks', [
     ]);
+
+    // Crypto Global key
+    await this.handleIndexesInCollection(Constants.DEFAULT_TENANT, 'keys', [
+      { fields: { key: 1 }, options: { unique: true } }
```

Comment:
Keep in mind that for crypto key change, you need to have access to both keys: the old and new ones.

In order to make that migration reliable and tested for each fields, I think you need to have a data structure like that one: 

+ clear text information (deciphered with the old key)
+ ciphered information with the old key
+ ciphered information with the new key

By doing so, you can conduct tests that the migration is successful for each fields and only store the ciphered information in DB if you sure that it match the clear text once deciphered with the new key.